src/util/recorder.ts

```diff
@@ -30,6 +30,8 @@ const MAX_BROWSER_TEXT_FETCH_SIZE = 25_000_000;
 
 const MAX_NETWORK_LOAD_SIZE = 200_000_000;
 
+const TAKE_STREAM_BUFF_SIZE = 1024 * 64;
```

Comment:
Might be good to make this configurable via args so people can fetch larger streams if they want?

Reply:
Oh, this isn't a limit, just size of buffer that is fetched at a time, streaming in 64k chunks. It is then either stored in memory or flushed to disk (if the the MAX_BROWSER_* total size is reached)

Reply:
Right, `BUFF_SIZE`! So the limit is just the appropriate timeouts?

Reply:
Those limits are for how much could be kept in memory before being buffered to disk. However, only content that is buffered in memory can be processed through the rewriting system and served back to browser. The limits for text are higher because it might be rewritten. There is no limit (beyond the disk volume size of course) for response that is streamed and buffered to disk, however, it is not served back to the browser. In case of a live-stream audio/video, the data is not actually played in the browser as a result during capture, but is written to disk, which should be fine (only issues might be for screenshot comparison...)

Probably should explain this better somewhere!